protocol/contracts/beanstalk/init/InitDistribution.sol

```diff
@@ -0,0 +1,49 @@
+/*
+ SPDX-License-Identifier: MIT
+*/
+
+pragma solidity ^0.8.20;
+
+import {AppStorage} from "contracts/beanstalk/AppStorage.sol";
+import {C} from "contracts/C.sol";
+import {LibDiamond} from "contracts/libraries/LibDiamond.sol";
```

Comment:
LibDiamond looks to be unused here, is that right?

Reply:
removed
https://github.com/BeanstalkFarms/Beanstalk/pull/860/commits/08614d4814895da7c59ddd8c24b0eb489ca783f0#diff-b0451bea09abb2293197eb00e388806cf4949fc7fd2952a7c926693780a93f60L9